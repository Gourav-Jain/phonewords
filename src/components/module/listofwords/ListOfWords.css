.bottom .result {
  text-align: center;
  color: #4747f3;
  padding: 5px;
}
ul {
  list-style: none;
  padding: 10px 20px;
  width: 300px;
  margin: 0 auto;
  color: black;
}
li {
  border-bottom: 1px solid lightgrey;
  padding: 10px;
}
li:last-child {
  border-bottom: none;
}
h4 {
  color: green;
  padding: 10px;
}
