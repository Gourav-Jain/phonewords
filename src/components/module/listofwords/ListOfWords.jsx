import React from "react";
import "./ListOfWords.css";

export const ListOfWords = ({ list }) => {
  return (
    <div className="result">
      <h4>List Of Words</h4>
      <ul>
        {list.map((key, val) => (
          <li key={val}>{key}</li>
        ))}
      </ul>
    </div>
  );
};

export default ListOfWords;
