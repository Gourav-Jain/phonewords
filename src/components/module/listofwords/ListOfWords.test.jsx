import React from "react";
import { render } from "@testing-library/react";
import ListOfWords from "./ListOfWords";

const mockList = ["ad", "ae", "af", "bd", "be", "bf", "cd", "ce", "cf"];
test("@ListOfWords", () => {
  const { container } = render(<ListOfWords list={mockList} />);
  expect(container).toMatchSnapshot();
});
