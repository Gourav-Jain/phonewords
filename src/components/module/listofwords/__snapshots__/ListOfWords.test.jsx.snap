// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@ListOfWords 1`] = `
<div>
  <div
    class="result"
  >
    <h4>
      List Of Words
    </h4>
    <ul>
      <li>
        ad
      </li>
      <li>
        ae
      </li>
      <li>
        af
      </li>
      <li>
        bd
      </li>
      <li>
        be
      </li>
      <li>
        bf
      </li>
      <li>
        cd
      </li>
      <li>
        ce
      </li>
      <li>
        cf
      </li>
    </ul>
  </div>
</div>
`;
